rtmbot/bin/run_rtmbot.py

```diff
@@ -18,11 +19,16 @@ def parse_args():
     )
     return parser.parse_args()
 
-# load args with config path
-args = parse_args()
-config = yaml.load(open(args.config or 'rtmbot.conf', 'r'))
-bot = client.init(config)
-try:
-    bot.start()
-except KeyboardInterrupt:
-    sys.exit(0)
+
+def main(args=None):
+    # load args with config path
+    args = parse_args()
+    config = yaml.load(open(args.config or 'rtmbot.conf', 'r'))
```

Comment:
Super tiny nitpick, feel free to ignore: what do you think about [using a context manager](https://docs.python.org/2/library/stdtypes.html#file.close) to open and read the config file?